ui/admin/app/styles/app.scss

```diff
@@ -987,3 +987,8 @@
 .add-policy-link {
   margin-bottom: 1.5rem;
 }
+
+// generic loading indicator
+.loading-indicator {
+  margin-top: 10rem;
```

Comment:
Could this have been a relative value?

Reply:
Oh yeah a relative number would be better. I will play around with that and update this to accommodate varying screen sizes.